nextflow.config

```diff
@@ -12,6 +12,23 @@ params {
     // Input options
     input                      = null
 
+    // Taxonomy classification options
+    taxonomy_mmseqs_classification_off      = false
+    taxonomy_mmseqs_save_intermedfiles      = false
+
+    taxonomy_mmseqs_databases_localpath     = null
```

Comment:
Same above, following other schema: `--taxonomic_classification_database`

And generally everywhere - `taxonomic_classification` rather than just `taxonomy`. Taxonomy means something different to bioinformaticians and they may get confused :grimacing:

Reply:
Im afraid that wont work because the param label would end up being 'taxonomic_classification_mmseqs_databases_localpath' for eaxmple. i have to specifiy which tool the param is coming from and in this case thats <mmseqs_database> and the arg itself <localpath>, sonow that we specified the tool for taxonomic classification and following the rest of the schema we have i would suggest 'mmseqs_databases_localpath' instead. This cuts short the param name and is informative. What do u think?